edx_sga/tests.py

```diff
@@ -197,6 +237,9 @@ def test_student_view_with_upload(self, Fragment, render_template):
     @mock.patch('edx_sga.sga.render_template')
     @mock.patch('edx_sga.sga.Fragment')
     def test_student_view_with_annotated(self, Fragment, render_template):
```

Comment:
pylint is calling out these as invalid names because parameters shouldn't have capital letters, you should change all these to `mock_fragment` or just `fragment`